projects/plugins/jetpack/modules/google-analytics/wp-google-analytics.php

```diff
@@ -54,11 +54,10 @@ private function __construct() {
 		// At this time, we only leverage universal analytics when enhanced ecommerce is selected and WooCommerce is active.
 		// Otherwise, don't bother emitting the tracking ID or fetching analytics.js
 		if ( class_exists( 'WooCommerce' ) && Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled() ) {
-			$analytics     = new Jetpack_Google_Analytics_Universal();
-			$amp_analytics = new Jetpack_Google_AMP_Analytics();
-
+			self::$analytics = new Jetpack_Google_Analytics_Universal();
+			$amp_analytics   = new Jetpack_Google_AMP_Analytics();
```

Comment:
Hey @douglas-johnson seems like the linter has a gripe about the unused `$amp_analytics` assignment. Could you change that to: `new Jetpack_Google_AMP_Analytics();`

Reply:
@samiff Sure, done.